tests/s2n_test.h

```diff
@@ -25,6 +25,8 @@
 #include "error/s2n_errno.h"
 #include "utils/s2n_safety.h"
 
+int test_count = 0;
```

Comment:
Because it's not in the global scope, we're limited in non-main() test functions like the one in s2n_handshake_test. We can't use any of the EXPECT_* functions designed for our tests, so we reimplement them all over the place. It also potentially makes debugging harder.

But that is a fair point -- I'll add count=0 to the BEGIN method, to reset the count before the official test, just in case.